src/sentry/api/serializers/rest_framework/environment.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import
+
+from rest_framework import serializers
+
+from sentry.models import Environment
+
+ValidationError = serializers.ValidationError
+
+
+class EnvironmentField(serializers.Field):
+    def to_representation(self, value):
+        return value
+
+    def to_internal_value(self, data):
+        try:
+            environment = Environment.objects.get(
+                organization_id=self.context["organization"].id, id=data
+            )
+        except Environment.DoesNotExist:
+            raise ValidationError("Environment is not part of this organization")
+        return environment.id
```

Comment:
May as well just return this as the environment. Then we can just pass the actual environments to the create/update methods.

Reply:
Okay. That should be cleaner.